rune/libenclave/attestation/registry.go

```diff
@@ -3,7 +3,7 @@ package attestation // import "github.com/opencontainers/runc/libenclave/attesta
 var registry []Registry
 
 type Registry interface {
-	Create(p map[string]string) (*Service, error)
+	Create(p map[string]string, raEnclaveType uint32) (*Service, error)
```

Comment:
Why not moving raEnclaveType into p (configuration)?

Reply:
Because raEnclaveType currently still belongs to enclave-specific concept. Hiding it under p (config)  is appropriate.

Reply:
> Why not moving raEnclaveType into p (configuration)?

Because the type of 'p' is `map[string]string` while the type of `raEnclaveType` is `uint32`. MoreOver, We transfered `raEnclaveType` from `string` to `uint32` before. So I think that in order to place `raEnclaveType` to 'p' to revert 'string' to 'uint32' is not necessary.